src/utils.js

```diff
@@ -73,6 +39,29 @@ export function getSides(sides?: string = ''): Array<IndividualSides> {
   return uniq(allSides)
 }
 
+export function getMarginClasses(
+  margin?: string,
+  marginSize?: MarginSizes
+): Array<string | void> {
+  const marginSizeClasses = {
```

Comment:
sounds like a problem for V8 to optimize, I'm happy to move it but I doubt that it makes any difference on runtime